### DISCUSSION TITLE
How to use chezmoi without dot syntax ?

### DISCUSSION THREAD

**Discussion by johackim** [twpayne/chezmoi#1821] [Q&A] [answered]
Hello,

This tools look like very great.

Can we use it without respect the dot syntax ?

Ex:

- Use `.tmux.conf` instead `dot_tmux.conf`
- Use `.vimrc` instead `dot_vimrc`
- Etc...

Thanks !

**Comment by twpayne** [comment, marked as answer]
tl;dr no, [this FAQ entry](https://github.com/twpayne/chezmoi/blob/master/docs/FAQ.md#can-i-change-how-chezmois-source-state-is-represented-on-disk) explains why.
